package.json

```diff
@@ -49,7 +49,7 @@
     "@babel/preset-react": "7.18.6",
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-replace": "^2.3.1",
-    "@stripe/connect-js": "3.3.7",
```

Comment:
The `peerDependencies` entry below should be bumped as well

Reply:
Thanks for catching that